pkg/cloud/cloud.go

```diff
@@ -125,11 +125,11 @@ func SetCustomPricing(obj *CustomPricing, name string, value string) error {
 		return fmt.Errorf("cannot set %s field value", name)
 	}
 
-	structFieldType := structFieldValue.Type()
+	// structFieldType := structFieldValue.Type()
```

Comment:
why change this?

Reply:
> why change this?

modify the logic of the validation to make it reasonable